vagrant/.gitignore

```diff
@@ -1 +1,3 @@
-.vagrant
\ No newline at end of file
+.vagrant
+*.box
+Berksfile.lock
```

Comment:
Also, ignoring the Berksfile.lock, etc., is handled in [`cookbook/.gitignore`](https://github.com/HubSpot/Baragon/blob/master/cookbook/.gitignore#L9-L12), so you can omit any Chef-y stuff in this `.gitignore`.